README.md

```diff
@@ -30,8 +30,8 @@ A deployment guide is available from the Fortinet Document Library:
 
 ## Launch a demo
 
-| New VPC, no-TGW | Existing VPC, no-TGW | TGW Integration (new VPC) |
-| ------ | ------ | ------|
+| New VPC, no-TGW                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Existing VPC, no-TGW                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | TGW Integration (new VPC)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
What happened here?

Reply:
> What happened here?

seems to be a formatter issue on the vscode. It changed the lines by adding extra spaces and dashes.

Reply:
does it look the same when you convert it?

Reply:
yes. it looks the same when displays on GitHun, just a bit annoying and excessively decorated as plain text because of the long line of a url.

turns out a vscode extension has caused this.